README.md

```diff
@@ -1,6 +1,9 @@
 # Protractor helper's library
 
 [![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper)
+[![npm](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper)
```

Comment:
What about calling it weekly downloads?

Reply:
I will change to 'npm weekly downloads'.

Reply:
Keep me posted when this is updated.

Reply:
@PauloGoncalvesBH, what about putting all the badges in the same line, like this:
`[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![npm](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier) [![Build Status](https://semaphoreci.com/api/v1/wlsf82/protractor-helper/branches/master/badge.svg)](https://semaphoreci.com/wlsf82/protractor-helper)`
It looks much nicer when rendered in the markdown.
Look:
<img width="684" alt="captura de tela 2018-11-17 as 17 48 16" src="https://user-images.githubusercontent.com/2768415/48663472-fc119100-ea90-11e8-867a-5a5c3ace0efe.png">